src/ripple/app/main/DBInit.cpp

```diff
@@ -131,6 +131,13 @@ const char* WalletDBInit[] =
         PrivateKey      CHARACTER(52)               \
     );",
 
+    // Peer reservations.
+    // REVIEWER: How do we handle table migrations if we need to add a column?
```

Comment:
Great question.  I don't know the answer.  But when you do find out the answer I think you should write up that answer as a comment.  Thanks.